flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/plan/stream/table/validation/TableAggregateValidationTest.scala

```diff
@@ -116,16 +116,16 @@ class TableAggregateValidationTest extends TableTestBase {
 
   @Test
   def testInvalidAliasWithWrongNumber(): Unit = {
+    val func = new EmptyTableAggFunc
     expectedException.expect(classOf[ValidationException])
     expectedException.expectMessage("List of column aliases must have same degree as " +
       "table; the returned table of function " +
-      "'org.apache.flink.table.planner.utils.EmptyTableAggFunc' has 2 columns, " +
+      s"'${func.functionIdentifier()}' has 2 columns, " +
```

Comment:
I think displaying a class name or a registered name is better than a functionIdentifier here. 
If we want to support this, I think we may need to refactor `TableAggregateFunctionDefinition` `AggregateFunctionDefinition`, etc... to have an `FunctionIdentifier` and a `displayName`.

Reply:
It's ok for me to stay the same

Reply:
It's okay for me to stay the same.